topydo/ui/TodoListWidget.py

```diff
@@ -240,6 +261,8 @@ def resolve_action(self, p_action_str, p_size=None):
                 self._execute_on_selected(cmd, execute_signal)
             else:
                 urwid.emit_signal(self, execute_signal, cmd)
+
+            urwid.emit_signal(self, 'save_cmd', cmd, execute_signal)
         else:
```

Comment:
Ok, I've followed my own second proposition. Results of this approach are visible in 1e1499e.